ngpvan-public-event-listing.gemspec

```diff
@@ -22,12 +22,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "faraday"
-  spec.add_dependency "faraday_middleware"
-  spec.add_dependency "mechanize"
+  spec.add_dependency 'faraday', '~> 0.15', '>= 0.15.3'
+  spec.add_dependency 'faraday_middleware', '~> 0.12', '>= 0.12.2'
+  spec.add_dependency 'mechanize', '~> 2.7', '>= 2.7.6'
 
-  spec.add_development_dependency "bundler", "~> 1.16"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency 'bundler', '~> 1.16'
+  spec.add_development_dependency 'rake', '~> 10.0'
+  spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'byebug'
```

Comment:
There's no `Gemfile` for this gem?

Reply:
[The gemfile](https://github.com/controlshift/ngpvan-public-event-listing/blob/master/Gemfile) points to the gemspec, unlike most of our gems where we build the gemspec from the Gemfile.